src/main/java/controllers/VisualizationController.java

```diff
@@ -0,0 +1,118 @@
+package controllers;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ComboBox;
+import javafx.util.StringConverter;
+import model.data.DataColumn;
+import model.data.DataModel;
+import model.data.DataTable;
+
+import java.util.Observable;
+import java.util.Observer;
+
+/**
+ * Control visualisation.
+ * Created by Chris on 26-5-2015.
+ */
+public class VisualizationController implements Observer {
+	@FXML
+	private ComboBox<String> visualization;
+	@FXML
+	private ComboBox<DataTable> table;
+	@FXML
+	private ComboBox<DataColumn> input1;
```

Comment:
ik bedoelde niet dat je ze moest renamen naar een globale naam. Iedere visualisatie heeft bepaalde dingen die hij moet instellen. Zo heeft deze twee columns nodig. De boxplot heeft slecht 1 column nodig. Een state transtition visualisatie heeft weer totaal andere dingen nodig. Dus de invoer velden voor een specifieke visualisatie moeten niet worden gedefinieerd in de algemene visualizationController